[Solved] npm start fails on Win 10

Installation of requirements works, 
however, start fails on Windows 10:

Looks like a unix -like environment with cp is assumed. Any idea what to fix and how? 

```
cp doesn't exist. build; doesn't exist. build doesn't exist
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! grand-stack-starter-api@0.0.1 build: `babel src -d build; cp .env build; cp src/schema.graphql build`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the grand-stack-starter-api@0.0.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\marvi\AppData\Roaming\npm-cache\_logs\2019-10-23T17_33_35_021Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! grand-stack-starter-api@0.0.1 start: `npm run build && node build/index.js`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the grand-stack-starter-api@0.0.1 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

--- a/api/package.json
+++ b/api/package.json
@@ -6,8 +6,8 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "start-dev": "./node_modules/.bin/nodemon --watch src --ext js,graphql --exec babel-node  src/index.js",
-    "build": "babel src -d build; cp .env build; cp src/schema.graphql build",
-    "now-build": "babel src -d build; cp .env build; cp src/schema.graphql build",
+    "build": "babel src -d build && shx cp .env build && shx cp src/schema.graphql build",
+    "now-build": "babel src -d build && shx cp .env build && shx cp src/schema.graphql build",
     "start": "npm run build && node build/index.js",
     "seedDb": "./node_modules/.bin/babel-node src/seed/seed-db.js"
   },
@@ -30,7 +30,8 @@
     "babel-preset-env": "^1.7.0",
     "husky": "^1.3.1",
     "lint-staged": "^8.2.1",
-    "nodemon": "^1.19.1"
+    "nodemon": "^1.19.1",
+    "shx": "^0.3.2"
   },
   "husky": {
     "hooks": {

